transports/noise/src/lib.rs

```diff
@@ -58,13 +58,17 @@
 mod io;
 mod protocol;
 
+use futures::future::BoxFuture;
 pub use io::Output;
```

Comment:
Nit: Let's keep this with the other, non-pub `use` please.

Reply:
:+1: